src/api_server/swagger/firecracker.yaml

```diff
@@ -328,9 +328,9 @@ paths:
           description: The MMDS configuration as JSON.
           required: true
           schema:
-            type: "#/definitions/MmdsConfig"
+            $ref: "#/definitions/MmdsConfig"
       responses:
-        204:
+        200:
```

Comment:
Shouldn't this be `204`? `PUT` requests return `NoContent` status when successful.

Reply:
You are right, it is actually a `204` NoContent response. Modified to `204`.